trello2moment

```diff
@@ -184,6 +184,23 @@ function fetchFile(url,fileName) {
   }
 }
 
+function createMomentDefinition(boardJSON, momentName) {
```

Comment:
This looks pretty good.  I would say just add two things.  First, use the background image as the image for this, so we have something to show on the page, and second, in addition to a DirectContainer, this should also be of type ucdlib:Moment,

Reply:
Updated!